core/spec.md

```diff
@@ -1906,30 +1906,56 @@ The following describes the attributes of Registry model:
         the xRegistry metadata - in other words JSON, or<br>
     2 - The Resource's document can be considered a "string" and therefore
         can be serialized as a "string" with some escaping.<br>
+
     For some well-known `contenttype` values (e.g. `application/json`) the
     first case can be easily determined by the server. However, for custom
     values the server will need to be explicitly told to when to consider it
     to be of the correct type. The `typemap` attribute allows for this by
     defining a mapping of `contenttype` values to well-known xRegistry format
     types.
 
+    Since the `contenttype` value is a "media-type" per
```

Comment:
Remembering adding this to the CloudEvents spec recently, maybe emphasize that this has to be done case-insensitively? I see this is mentioned later on, but it feels useful to have everything to do with lookup specified in one place.

Reply:
done